nano/node/vote_processor.cpp

```diff
@@ -241,28 +241,34 @@ bool nano::vote_processor::empty () const
 void nano::vote_processor::calculate_weights ()
 {
 	nano::unique_lock<nano::mutex> lock{ mutex };
-	if (!stopped)
+
+	if (stopped)
+	{
+		return;
+	}
+
+	representatives_1.clear ();
+	representatives_2.clear ();
+	representatives_3.clear ();
+
+	auto online = online_reps.online ();
```

Comment:
I don't think we want to change this from online_reps.trended (). Trended has hysteresis if the network is partitioned or the node is disconnected but online_reps.online doesn't, and might be 0 if the node is disconnected.

Reply:
Okay, my goal was for it to be able to react to rep availability changes faster, but thinking about it now, trended makes more sense.